ONSdigital/census-rm-docker-dev
# Add enriched fulfilment queue, exchange and binding

# Motivation and Context
If Gov Notify is down then we will end up creating bazillions of UAC-QID pairs, because we create a pair every time we attempt to process a fulfilment message and we don't roll back the transaction, because it's not participating in a distributed transaction.

# What has changed
Added queue and binding.

# How to test?
Run the acceptance tests along with Notify Processor on branch `enrich-fulfilment-message-with-uac`. There should be zero regression. Try stopping the Notify Stub. Check that bazillions of UAC-QID pairs are not created.

# Links
Trello: https://trello.com/c/9rrv68nL
